### ISSUE TITLE
FlexSlider2 support feature

### ISSUE THREAD

**Issue by thezed** [priviterag/spree_slider#12]
I found that Nivo and Anything sliders are not responsible for window resizing and it would be great for this gem to support some such as FlexSlider2.

BTW Good job with the gem, keep it going

**Comment by JDutil** [comment]
I think this extension should move away from supporting multiple carousel libraries, and just promote rolling your own as referenced in the README.

**Comment by joshuapaling** [comment]
Same - I think it should just support rolling your own.

**Comment by JDutil** [comment]
@priviterag this extension may be useful over at github.com/spree-contrib see my blog post http://spreecommerce.com/blog/the-state-of-extensions-2014 although I may be biased for using this in my website.

**Comment by joshuapaling** [comment]
Yeah I think a slider plugin is a pretty common requirement. I think this'd be good in spree-contrib

**Comment by priviterag** [comment]
Hi guys,

I'm glad to see that you find it useful. I need to find some spare time to
publish it on spree-contrib or, if you have time, please, send me a pull
request with the changes required and I'll merge it.

Kind regards,
Giuseppe

**Comment by joshuapaling** [comment]
@JDutil Your blog post doesn't actually say who to contact in order to get a project on spree-contrib. But if it does get added, it could be worth adding this fork: https://github.com/zewelor/spree_slider which has a couple of extra commits worth including.

**Comment by JDutil** [comment]
@priviterag you should be able to simply transfer the repo, and I can add you as a contributor.  Github automatically sets up redirects, and allows people to continue bundling even without updating their Gemfiles.

**Comment by priviterag** [comment]
@JDutil tried to transfer to spree-contrib. it seems I need administrative access to the organisation to do it

**Comment by JDutil** [comment]
Odd I thought it worked for someone else, but maybe I just created them a repo and had them push to it.  If you want to add me as an admin to this repo I could do it or we could just create a repo for you to push to, but that wouldn't setup github redirects...

**Comment by joshuapaling** [comment]
@priviterag Any chance you could add @JDutil as an admin, and we can transfer this over to spree-contrib? Otherwise, @JDutil could you just fork this repo, and we'll do without the redirects?

**Comment by priviterag** [comment]
@joshuapaling @JDutil seems good. going to add jdutil

**Comment by joshuapaling** [comment]
Thanks! And thanks for this plugin, it's great.

**Comment by priviterag** [comment]
@joshuapaling  I cannot add @JDutil as administrator. not sure if it will work. let me know guys!

**Comment by JDutil** [comment]
@priviterag I've created https://github.com/spree-contrib/spree_slider and added you as a committer.  If there is anyone else that should be added to the repo let me know.

**Comment by priviterag** [comment]
@JDutil Thanks Jeff!
